controllers/controllers.js

```diff
@@ -0,0 +1,9 @@
+const { selectCategories } = require("../models/models");
```

Comment:
You might want to rename your current model and categories file to be more semantic (e.g. categoriesController/categoriesModel). Not an issue now, but something to watch out for later.